src/Symfony/Bridge/Twig/Mime/WrappedTemplatedEmail.php

```diff
@@ -42,14 +42,16 @@ public function toName(): string
      *                                 some Twig namespace for email images (e.g. '@email/images/logo.png').
      * @param string|null $contentType The media type (i.e. MIME type) of the image file (e.g. 'image/png').
      *                                 Some email clients require this to display embedded images.
+     * @param string|null $name        A custom file name that overrides the original name (filepath) of the image
```

Comment:
Adding a new parameter is a new feature, which should go in 7.3 rather than in 6.4. Please separate that from the bugfix.